oracle/networks.py

```diff
@@ -138,6 +146,14 @@
             "0x0100000000000000000000005c631621b897f467dd6a91855a0bc97d77b78dc0"
         ),
         ORACLE_PRIVATE_KEY=config("ORACLE_PRIVATE_KEY", default=""),
+        ORACLE_STAKEWISE_OPERATOR=Web3.toChecksumAddress(
+            "0xa11c358c20b2baa682956ea2c7a659f6e9583f96"
```

Comment:
Lets set only mainnet operator, others to zero address, we can remove ORACLE_IGNORE_STAKEWISE_OPERATOR flag then